Documentation/contributing.rst

```diff
@@ -812,303 +960,370 @@ Have a question about how the tests work or want to chat more about improving th
 
-11. Confirm that you want to disable the build trigger.
+::
 
-.. image:: images/cilium-quayio-tag-7.png
-    :width: 600px
-    :align: center
-    :height: 300px
+    $ make render-docs
 
-12. Once the build is finished click under Tags (on the left menu).
-13. Click on the wheel and;
-14. Add a new tag to the image that was built.
+This builds the docs in a container and builds and starts a web server with
+your document changes.
 
-.. image:: images/cilium-quayio-tag-8.png
-    :width: 600px
-    :align: center
-    :height: 300px
+Now the documentation page should be browsable on http://localhost:8080.
 
-15. Write the name of the tag that you want to give for the newly built image.
-16. Confirm the name is correct and click on "Create Tag".
 
-.. image:: images/cilium-quayio-tag-9.png
-    :width: 600px
-    :align: center
-    :height: 300px
 
-17. After the new tag was created you can delete the other tag, which is the
-    name of your branch. Select the tag name.
-18. Click in Actions.
-19. Click in "Delete Tags".
+CI / Jenkins
+------------
 
-.. image:: images/cilium-quayio-tag-10.png
-    :width: 600px
-    :align: center
-    :height: 300px
+The main CI infrastructure is maintained at https://jenkins.cilium.io/
 
-20. Confirm that you want to delete tag with your branch name.
+Jobs Overview
+~~~~~~~~~~~~~
 
-.. image:: images/cilium-quayio-tag-11.png
-    :width: 600px
-    :align: center
-    :height: 300px
+Cilium-PR-Ginkgo-Tests-Validated
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You have created a new image build with a new tag. The next steps should be to
-update the repository root's Dockerfile so that it points to the new
-``cilium-builder`` or ``cilium-runtime`` image recently created.
+Runs validated Ginkgo tests which are confirmed to be stable and have been
+verified. These tests must always pass.
 
+The configuration for this job is contained within ``ginkgo.Jenkinsfile``.
 
-Submitting a pull request
--------------------------
+It first runs unit tests using docker-compose using a YAML located at
+``test/docker-compose.yaml``.
 
-Contributions may be submitted in the form of pull requests against the
-github repository at: `<https://github.com/cilium/cilium>`_
+The next steps happens in parallel:
 
-Before hitting the submit button, please make sure that the following
-requirements have been met:
+    - Runs the runtime e2e tests.
+    - Runs the Kubernetes e2e tests against the latest default version of Kubernetes specified above.
+
+
+Cilium-PR-Ginkgo-Tests-k8s
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Runs the Kubernetes e2e tests against all Kubernetes versions that are not
+currently not tested as part of each pull-request, but which Cilium still
+supports, as well as the the most-recently-released versions of Kubernetes that
+are not yet declared stable by Kubernetes upstream:
+
+First stage (stable versions which Cilium still supports):
+
+    - 1.7
+    - 1.8
+
```

Comment:
Add 1.9

Reply:
Fixed to 1.7 and 1.10 as this is what `ginkgo.Jenkinsfile` currently lists

Reply:
Fixed to 1.7 and 1.10 as per `ginkgo.Jenkinsfile`